src/tests/ftest/daos_test/daos_core_test-rebuild.py

```diff
@@ -32,6 +32,13 @@ class DaosCoreTestRebuild(DaosCoreBase):
     :avocado: recursive
     """
 
+    # Test variants that should be skipped
+    CANCEL_FOR_TICKET = [
+        ["DAOS-5851", "test_name", "rebuild tests 0-10"]
+        ["DAOS-5851", "test_name", "rebuild tests 0-10"],
```

Comment:
(pylint-invalid-sequence-index) Sequence index is not an int, slice, or instance with __index__